models/iterable__events.sql

```diff
@@ -14,7 +14,10 @@ with events as (
     from {{ var('event') }}
 
     {% if is_incremental() %}
-    where created_at >= (select max(created_at) from {{ this }} )
+    where created_on >= {{ iterable.iterable_lookback(
+        from_date="max(created_on)", 
+        interval=var('lookback_window', 7), 
+        datepart='day') }}
```

Comment:
Also small request, but can you order the parameters in the same order they are when defining the macro. So it should go: (`from_date`, `datepart`, `interval`)

Reply:
done!